idd/Energy+.idd.in

```diff
@@ -39706,56 +39745,60 @@ AirTerminal:SingleDuct:ConstantVolume:CooledBeam,
+AirTerminal:SingleDuct:Mixer,
+       \memo The mixer air terminal unit provides a means of supplying central system
+       \memo air to the air inlet or outlet side of a zoneHAC equipment such as a four
+       \memo pipe fan coil unit. Normally the central air would be ventilation air from
+       \memo a dedicated outdoor air system (DOAS).
    A1, \field Name
        \required-field
-   A2, \field ZoneHVAC Terminal Unit Object Type
+   A2, \field ZoneHVAC Unit Object Type
        \required-field
        \type choice
        \key ZoneHVAC:FourPipeFanCoil
        \key ZoneHVAC:WaterToAirHeatPump
-   A3, \field ZoneHVAC Terminal Unit Name
+       \key ZoneHVAC:PackagedTerminalAirConditioner
+       \key ZoneHVAC:PackagedTerminalHeatPump
+       \key ZoneHVAC:TerminalUnit:VariableRefrigerantFlow
+       \key AirLoopHVAC:UnitarySystem
+       \note The type of ZoneHVAC equipment to which this terminal mixer will be connected.   
+   A3, \field ZoneHVAC Unit Object Name
        \required-field
        \type object-list
        \object-list DOAToZonalUnit
-   A4, \field Terminal Unit Outlet Node Name
+       \note The name of ZoneHVAC equipment to which this terminal mixer will be connected.   
+   A4, \field Mixer Outlet Node Name
        \required-field
        \type node
-   A5, \field Terminal Unit Primary Air Inlet Node Name
+       \note This is the outlet air node name of the mixer. This will be an inlet air node name 
+       \node of the ZoneHVAC equipment if the connection type in the input field Mixer Connection 
+       \note Type below is InletSide, else this will be inlet air node name of the conditioned zone
+       \node if the connection type in the input field Mixer Connection Type below is SupplySide.
+   A5, \field Mixer Primary Air Inlet Node Name
        \required-field
        \type node
-   A6; \field Terminal Unit Secondary Air Inlet Node Name
+       \note The primary air (treated outdoor air) inlet node name of the mixer. This will be an 
+       \note outlet air node name of an AirLoopHVAC:ZoneSplitter, providing the connection to the 
+       \note DOAS system. 
+   A6, \field Mixer Secondary Air Inlet Node Name
        \required-field
        \type node
-
-AirTerminal:SingleDuct:SupplySideMixer,
-       \memo The supply side mixer air terminal unit provides a means of supplying central system air
-       \memo to the air outlet of a zone AC unit such as a four pipe fan coil. Normally the central air
-       \memo would be ventilation air from a dedicated outdoor air system (DOAS).
-   A1, \field Name
-       \required-field
-   A2, \field ZoneHVAC Terminal Unit Object Type
+       \note The secondary air (recirculating air) inlet node name of the mixer. This will be 
+       \note the outlet air node name of the ZoneHVAC equipment if the connection type in the 
+       \note input field mixer Connection Type below is SupplySide, or else this will be exhaust
```

Comment:
this will be *an* exhaust . . .
I realize these "the" and "an" changes seem picky, but these connections can be confusing, so making them as clear as possible is helpful.  In some cases it must be an exact match with a node name in the ZoneHVAC equipment, but in others, it's an arbitrary node name that also needs to be a zone inlet or zone exhaust.